app/domain/logs.rb

```diff
@@ -389,6 +389,11 @@ module AuthnJwt
         msg: "Fetching jwks-uri signing key",
         code: "CONJ00076D"
       )
+
+      FetchingTokenString = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching JWT token from request",
+        code: "CONJ00077D"
```

Comment:
Do we have "Fetched" msg?
according to this: https://ca-il-confluence.il.cyber-ark.com/display/rndp/Message+Pool#/